src/components/ResidentDocumentsTable.tsx

```diff
@@ -0,0 +1,285 @@
+import React, { FunctionComponent, useMemo, useState } from 'react';
+import { EvidenceList, EvidenceTile } from './EvidenceTile';
+import { formatDate } from '../helpers/formatters';
+import { EvidenceAwaitingSubmissionTile } from './EvidenceAwaitingSubmissionTile';
+import { EvidenceRequest } from '../domain/evidence-request';
+import { DateTime } from 'luxon';
+import {
+  DocumentSubmission,
+  IDocumentSubmission,
+} from '../domain/document-submission';
+import { DocumentType } from '../domain/document-type';
+import styles from '../styles/EvidenceTile.module.scss';
+
+type EvidenceAwaitingSubmission = {
+  documentType: string;
+  dateRequested: DateTime | undefined;
+  requestedBy: string | undefined;
+  reason: string | undefined;
+  kind: 'EvidenceAwaitingSubmission';
```

Comment:
These 'kinds' were added to the types so that we could discriminate on displaying certain components based on the types.